app/i18n/en/namespaces/common.js

```diff
@@ -15,6 +15,16 @@ const common = {
     edit: 'Edit',
     remove: 'Remove',
   },
+  link: {
+    topics: 'Topics',
+    newtopic: 'Create new topic',
+    library: 'Library',
+  },
+  feed: {
+    comment: 'commented on topic',
```

Comment:
Can we use i18n with interpolation/variables here? Word order might vary in different languages.

Reply:
Also, strings related to topics and feed don't really belong in the 'common' namespace, which is intended (as the name says) for commen elements between all pages / modules.